app/assets/stylesheets/custom.css.scss

```diff
@@ -26,11 +26,63 @@ body {
   padding: 5px;
 }
 
-.hero-unit {
+.hero-unit1 {
   background-image: image-url("volunteer-couple-hero.png");
-  background-size:cover;
-  height:600px;
+  background-size: cover;
+  filter: gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.